forge-game/src/main/java/forge/game/card/CardProperty.java

```diff
@@ -143,6 +143,10 @@ public static boolean cardHasProperty(Card card, String property, Player sourceC
             if (!card.isBackSide()) {
                 return false;
             }
+        } else if (property.equals("Transformed")) {
+            if (!card.isBackSide() || !card.canTransform()) {
```

Comment:
I think just checking `getCurrentStateName() == CardStateName.Transformed` might be more precise

Reply:
Sounds good to me!